raiden/api/v1/encoding.py

```diff
@@ -35,16 +35,15 @@
 )
 
 
-# type converter for the flask route
 class HexAddressConverter(BaseConverter):
-
     def to_python(self, value):
-        value = address_decoder(value)
-        return value
+        if value[:2] == '0x':
```

Comment:
Okay as discussed between the team we are a go with changing this but the change here will not work. In fact we should remove this HexAdddressConverter as it is not used at all.

Please remove it from api/rest.py and all the other places where it is located.